datastore/src/main/java/io/spine/server/storage/datastore/DsProperties.java

```diff
@@ -67,10 +93,51 @@ static boolean isDeleted(Entity entity) {
         return hasFlag(entity, LifecycleFlagField.deleted.toString());
     }
 
+    static OrderBy byCreatedTime() {
+        return AggregateEventRecordOrdering.BY_CREATED_PROPERTY.getOrdering();
+    }
+
+    static OrderBy byVersion() {
+        return AggregateEventRecordOrdering.BY_VERSION_PROPERTY.getOrdering();
+    }
+
+    static OrderBy byRecordType() {
+        return AggregateEventRecordOrdering.BY_SNAPSHOT_PROPERTY.getOrdering();
+    }
+
     private static boolean hasFlag(Entity entity, String flagName) {
         final boolean result =
                 entity.contains(flagName)
                         && entity.getBoolean(flagName);
         return result;
     }
+
+    private enum AggregateEventRecordProperty {
```

Comment:
In Java 8 we would automatically get lowercasing by having default `toString()` impl.